docs/release_notes.rst

```diff
@@ -11,24 +11,25 @@ Supported Platforms
 *RTI Connector* works with Python® 2.x and 3.x. It uses a native C library that
 runs on most Windows®, Linux® and macOS® platforms.
 
-*Connector* has been tested with Python 2.6+ and 3.6.8+, and on the following systems:
-     
+*Connector* has been tested with Python 2.6+ and 3.6.8+, on the following systems:
```

Comment:
I think this was correct with the "and". We don't test with both versions of Python on every system below, we just test on all of those systems and also we have tested both listed versions of Python

Reply:
I fixed it